pkg/ttnpb/udp/packet_data.go

```diff
@@ -40,7 +40,7 @@ type RxPacket struct {
 	Modu string       `json:"modu"`           // Modulation identifier "LORA", "FSK" or "LR-FHSS"
 	DatR datarate.DR  `json:"datr"`           // LoRa datarate, FSK datarate or LR-FHSS datarate
 	CodR string       `json:"codr"`           // LoRa or LR-FHSS ECC coding rate identifier
-	Hpw  string       `json:"hpw"`            // LR-FHSS hopping channel width (grid), in kHz
+	Hpw  uint32       `json:"hpw"`            // Hopping width; a number describing the number of steps of the LR-FHSS grid
```

Comment:
```suggestion
	HPW  uint32       `json:"hpw,omitempty"`  // Hopping width; a number describing the number of steps of the LR-FHSS grid
```

Reply:
`0` is an invalid number, it's only `8` or `52` (for now), and it's not always given.